README.md

```diff
@@ -41,4 +41,10 @@ npx hardhat test
 ### Forge
 ```bash
 forge test
+```
+
+## Deploy
+
+```bash
+NETWORK=<target_network> npx hardhat deploy
```

Comment:
my bad. This is actually `HARDHAT_NETWORK` and not `NETWORK`

Reply:
Done